Dockerfile

```diff
@@ -4,7 +4,7 @@ MAINTAINER Conjur Inc
 RUN apt-get update && apt-get install -y jq less 
 RUN go get -u github.com/jstemmer/go-junit-report
 RUN go get -u github.com/golang/dep/cmd/dep
-RUN go get github.com/smartystreets/goconvey
+RUN go get github.com/playscale/goconvey
```

Comment:
Why the change here?

Reply:
This is because I wanted to use `-tags` for enabling tests, oss vs enterprise. smartystreets/goconvey doesn't support tags, doesn't look like there has been any development on the repo since February 2018.

I found the playscale repo via this PR, looks like they are keeping goconvey more up to date, with support for tags

https://github.com/smartystreets/goconvey/pull/458

Reply:
Ok, good to know.